buildSrc/build.gradle

```diff
@@ -36,4 +36,6 @@ dependencies {
     implementation "io.codechicken:DiffPatch:${gradle.parent.ext.diffpatch_version}"
 
     implementation "org.ow2.asm:asm:${gradle.parent.ext.asm_version}"
+
+    implementation "org.apache.maven:maven-artifact:3.8.5"
```

Comment:
Is there a chance we might want to ever upgrade this? If yes it should likely use the same `gradle.parent.ext` trick.

Reply:
I would like a consistent place where all version numbers are defined so they are easily to lookup when somebody asks about them.

Is there any good reason to not move them to the same place as all the others?